pkg/quota/image/usage.go

```diff
@@ -314,39 +321,34 @@ func (c *GenericImageStreamUsageComputer) getImageReferenceForObjectReference(
 
 // getImageStream gets an image stream object from etcd and caches the result for the following queries.
 func (c *GenericImageStreamUsageComputer) getImageStream(namespace, name string) (*imageapi.ImageStream, error) {
-	key := fmt.Sprintf("%s/%s", namespace, name)
-	if is, exists := c.imageStreamCache[key]; exists {
-		return is, nil
+	isObject, exists, _ := c.imageStreamCache.GetByKey(fmt.Sprintf("%s/%s", namespace, name))
```

Comment:
The convention is usually to create a holder object and pass it in (if
you're using a reflector).

On Wed, Mar 23, 2016 at 4:48 AM, Michal Minar <notifications@github.com>
wrote:

> In pkg/quota/image/usage.go
> <https://github.com/openshift/origin/pull/8195#discussion_r57125237>:
>
> > @@ -314,39 +321,34 @@ func (c *GenericImageStreamUsageComputer) getImageReferenceForObjectReference(
> >
> >  // getImageStream gets an image stream object from etcd and caches the result for the following queries.
> >  func (c *GenericImageStreamUsageComputer) getImageStream(namespace, name string) (*imageapi.ImageStream, error) {
> > -	key := fmt.Sprintf("%s/%s", namespace, name)
> > -	if is, exists := c.imageStreamCache[key]; exists {
> > -		return is, nil
> > +	isObject, exists, _ := c.imageStreamCache.GetByKey(fmt.Sprintf("%s/%s", namespace, name))
>
> cache.MetaNamespaceKeyFunc() needs an object. I'll use ktypes.NamespacedName{namespace,
> name}.String() instead.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8195/files/d9f70e84dc18bde5a6518ac604b8fbc862e67725..56bf992b580cca4a97099df42a37bc921762e684#r57125237>
>